src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  flex-shrink: 0;
+}
+
+.nav__list {
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__item {
+  position: relative;
+}
+
+.nav__link {
+  color: #000;
+  display: block;
+  font-weight: 500;
+  line-height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
```

Comment:
Ensure that the CSS selector `[data-qa_hover]` matches the HTML attribute `data-qa="hover"`. Update the selector to `[data-qa="hover"]::after` to ensure the styles are applied correctly.